frontend/src/containers/Search/actions.js

```diff
@@ -0,0 +1,37 @@
+export const SEARCH_HOUSES_REQUEST = 'SEARCH_HOUSES';
+export function searchHouses(postCode='') {
+  return {
+    type: SEARCH_HOUSES_REQUEST,
+    postCode
+  };
+}
+
+export const SEARCH_HOUSES_SUCCESS = 'SEARCH_HOUSES_SUCCESS';
+export function searchHousesSuccess(postCode, json) {
+  return {
+    type: SEARCH_HOUSES_SUCCESS,
+    postCode,
+    houses: (json.houses || [])
```

Comment:
I don't think actions should manipulate data? Is this because you're just doing the filtering locally for now? shouldn't it be in the reducer anyway? These are more questions than opinion, really not confident with redux myself yet.

Reply:
I read couple of articles and it made some sense to me i.e. "Action can be fat but reducers should be lean", the rationale was reducers are just pure functions and it should not be responsible for business logic.

Reply:
I'm only learning redux, but I thought that was exactly the opposite way round? From the redux dox:
```
Reducers specify how the application's state changes in response to actions sent to the store. Remember that actions only describe the fact that something happened, but don't describe how the application's state changes.
```

So I'm completely confused now ... can you post a link to the articles you have read?

Reply:
Hmm, makes sense. I must have read it somewhere, too much to digest I will research it more.